backend/src/data/models/appointment.ts

```diff
@@ -0,0 +1,34 @@
+import { Sequelize, DataTypes, ModelCtor, Model } from 'sequelize';
+
+enum Type {
+  ONLINE = 'online',
+  OFFLINE = 'offline'
+}
```

Comment:
```suggestion
enum Type {
  ONLINE = 'online',
  OFFLINE = 'offline'
}
```
We'll need this enum for all apps, so it would be better to move it to the shared folder and change the name to more specific. Something like `AppointmentType`